app/src/main/java/org/p2p/wallet/auth/interactor/MetadataInteractor.kt

```diff
@@ -33,9 +33,9 @@ class MetadataInteractor(
             saveMetadataToStorage(value)
         }
 
-    suspend fun tryLoadAndSaveMetadata() {
+    suspend fun tryLoadAndSaveMetadata(): Boolean {
```

Comment:
uh, maybe you can create an inline class or wrapper class?
just returning Boolean is bad

Reply:
do you mean return some state?